src/Sylius/Bundle/ApiBundle/StateProvider/Shop/Payment/Request/ItemProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\StateProvider\Shop\Payment\Request;
```

Comment:
```suggestion
namespace Sylius\Bundle\ApiBundle\StateProvider\Shop\PaymentRequest;
```

Request is not a resource, so keeping `PaymentRequest` is maybe better here.